url.bs

```diff
@@ -1,26 +1,12 @@
 <pre class=metadata>
-Title: URL Standard
 Group: WHATWG
 H1: URL
 Shortname: url
-Status: LS
-No Editor: true
+Text Macro: TWITTER urlstandard
 Abstract: The URL Standard defines URLs, domains, IP addresses, the <code title>application/x-www-form-urlencoded</code> format, and their API.
-Logo: https://resources.whatwg.org/logo-url.svg
-Boilerplate: omit feedback-header, omit conformance
-!Participate: <a href=https://github.com/whatwg/url>GitHub whatwg/url</a> (<a href=https://github.com/whatwg/url/issues/new>new issue</a>, <a href="https://github.com/whatwg/url/issues">open issues</a>)
-!Participate: <a href="https://wiki.whatwg.org/wiki/IRC">IRC: #whatwg on Freenode</a>
-!Commits: <a href="https://github.com/whatwg/url/commits">https://github.com/whatwg/url/commits</a>
-!Commits: [SNAPSHOT-LINK]
-!Commits: <a href="https://twitter.com/urlstandard">@urlstandard</a>
-!Tests: <a href=https://github.com/w3c/web-platform-tests/tree/master/url>web-platform-tests url/</a> (<a href=https://github.com/w3c/web-platform-tests/labels/url>ongoing work</a>)
-!Translation (non-normative): <span title=Japanese><a href=https://triple-underscore.github.io/URL-ja.html lang=ja hreflang=ja rel=alternate>日本語</a></span>
+Translation: ja https://triple-underscore.github.io/URL-ja.html
```

Comment:
I'm pretty sure this does not generate the desired markup. That's why I use `!Translation (non-normative)` throughout.

Reply:
Check out the result; it seemed identical to me. But we can also fix Bikeshed while we're working on this.

Reply:
I think we should at least change the message to just say "non-normative". "Likely out-of-date" is not friendly to the translators. Is the generated output available somewhere? https://whatpr.org/url/360.html has a very different heading, though I suspect that's due to a Bikeshed mismatch.

Reply:
In https://github.com/tabatkins/bikeshed/pull/1155 I changed it to "potentially". Is that good or do you prefer just removing it?

Reply:
Here is the output:

Reply:
Here is the output:

Reply:
I prefer just removing it.

Reply:
OK, I'll update the Bikeshed PR; hopefully @tabatkins will be OK with it since I think it's mostly (all?) WHATWG specs that use this.

Reply:
I think I was waiting on https://github.com/tabatkins/bikeshed/issues/1058 to be fixed, but then forgot about it or maybe I was waiting for us to use our own templating since it didn't quite work out, even if that were fixed.